src/App.jsx

```diff
@@ -17,6 +18,7 @@ function App() {
     <NavigationBar/>
     <Routes>
       <Route path ="/" element={<ArticlesPage/>}/>
+      <Route path="/:article_id" element={<ArticleByIdPage/>}/>
```

Comment:
Right now the URL doesn't give any hints as to what the parametric bit is actually for - it would be a lot better for the url to come in a format of /articles/:article_id, since that lets the user know that the number is for an article

Reply:
good point - have updated! thank you!